pages/monbar.php

```diff
@@ -73,15 +76,96 @@
         <button>
             <img id="btnAbout" src="../ressources/images/info.svg" alt="Question Mark" width="24" height="24">
         </button>
-        <button>
+        <button type="button" id="publish" data-bs-toggle="modal" data-bs-target="#monModal">
             <img id="btnPublish" src="../ressources/images/feather.svg" alt="Feather" width="24" height="24">
```

Comment:
L'attribut `alt` doit décrire l'objectif ou la fonction du bouton plutôt que l'image elle-même.
Les attributs CSS devraient être dans un fichier CSS.
```suggestion
            <img id="btnPublish" src="../ressources/images/feather.svg" alt="Publier">
```